rocketgate-sdk/service/service.go

```diff
@@ -284,7 +284,7 @@ func (r GatewayService) getConnectTimeout(req *request.GatewayRequest) int64 {
 	return int64(connectTimeout) * 1000
 }
 
-func (r GatewayService) getServerNameAndCleanFailedParams(req *request.GatewayRequest, resp *response.GatewayResponse) (string, bool) {
+func (r *GatewayService) getServerNameAndCleanFailedParams(req *request.GatewayRequest, resp *response.GatewayResponse) (string, bool) {
```

Comment:
I don't think we need change to pointer for getServerNameAndCleanFailedParams (my test can pass without this change)? I thought pointer is mainly for update ?

Reply:
i probably left this in while i was trying to find the original bug, ill remove it